src/views/NotFound.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="main">
+    <pageHeader :voltar="true" />
+    <div class="columns is-marginless">
+        <div v-if="this.windowWidth > 768" class="column is-10 is-offset-1">
```

Comment:
aqui não precisa do `this`

Reply:
Feito! Há um PR dois pois uma DIV não estava fechada e deu erro no botão voltar.